dist/s_define.list

```diff
@@ -14,7 +14,11 @@ CONNECTION_CONTROL_POINT_DEFINE_RET
 CONNECTION_CONTROL_POINT_DEFINE_SLEEP
 CONNECTION_CONTROL_POINT_DEFINE_START
 CONNECTION_CONTROL_POINT_DEFINE_WAIT_FOR_TRIGGER
-CONNECTION_CONTROL_POINT_WAIT_FOR_TRIGGER
+CONNECTION_CONTROL_POINT_DO_WAIT_FOR_TRIGGER
+CONNECTION_CONTROL_POINT_SET_MATCH_VALUE_AND_DO_WAIT_FOR_TRIGGER
```

Comment:
This would be:
`CONNECTION_CONTROL_POINT_SET_MATCH_VALUE_AND_WAIT`
`CONNECTION_CONTROL_POINT_SET_TEST_VALUE_AND_TRIGGER`